2021-04-09 10:24:32,188 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting env: {}
2021-04-09 10:24:32,188 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-09 10:24:32,188 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '4e022d9a965c0e513287d998fbd6fd1d46056300'}
2021-04-09 10:24:32,188 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-09 10:24:32,188 INFO    MainThread:921080 [wandb_setup.py:_flush():69] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f456829c4c0>
2021-04-09 10:24:32,189 INFO    MainThread:921080 [wandb_init.py:_log_setup():319] Logging user logs to /home/ubuntu/evgeny/GEP-forecasts/EVERGI/wandb/run-20210409_102432-1xm392uh/logs/debug.log
2021-04-09 10:24:32,189 INFO    MainThread:921080 [wandb_init.py:_log_setup():320] Logging internal logs to /home/ubuntu/evgeny/GEP-forecasts/EVERGI/wandb/run-20210409_102432-1xm392uh/logs/debug-internal.log
2021-04-09 10:24:32,189 INFO    MainThread:921080 [wandb_init.py:init():352] calling init triggers
2021-04-09 10:24:32,189 INFO    MainThread:921080 [wandb_init.py:init():357] wandb.init called with sweep_config: {}
config: {'batch_size': 1500, 'architecture': 'RNN with forward lags for temporal', 'dataset': 'Columbia', 'epochs': 100, 'patience': 10}
2021-04-09 10:24:32,189 INFO    MainThread:921080 [wandb_init.py:init():401] starting backend
2021-04-09 10:24:32,189 INFO    MainThread:921080 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-09 10:24:32,422 INFO    MainThread:921080 [backend.py:ensure_launched():123] starting backend process...
2021-04-09 10:24:32,601 INFO    MainThread:921080 [backend.py:ensure_launched():127] started backend process with pid: 934009
2021-04-09 10:24:32,604 INFO    MainThread:921080 [wandb_init.py:init():406] backend started and connected
2021-04-09 10:24:32,741 INFO    MainThread:921080 [wandb_init.py:init():446] updated telemetry
2021-04-09 10:24:32,742 INFO    MainThread:921080 [wandb_init.py:init():465] communicating current version
2021-04-09 10:24:33,776 INFO    MainThread:921080 [wandb_init.py:init():470] got version response upgrade_message: "wandb version 0.10.25 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-09 10:24:33,777 INFO    MainThread:921080 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-04-09 10:24:34,091 INFO    MainThread:921080 [wandb_init.py:init():503] starting run threads in backend
2021-04-09 10:24:35,760 INFO    MainThread:921080 [wandb_run.py:_console_start():1422] atexit reg
2021-04-09 10:24:35,760 INFO    MainThread:921080 [wandb_run.py:_redirect():1285] redirect: SettingsConsole.WRAP
2021-04-09 10:24:35,761 INFO    MainThread:921080 [wandb_run.py:_redirect():1320] Wrapping output streams.
2021-04-09 10:24:35,761 INFO    MainThread:921080 [wandb_run.py:_redirect():1336] Redirects installed.
2021-04-09 10:24:35,761 INFO    MainThread:921080 [wandb_init.py:init():527] run started, returning control to user process
2021-04-09 10:50:28,986 INFO    MainThread:921080 [wandb_init.py:_pause_backend():240] pausing backend
2021-04-09 10:58:46,150 INFO    MainThread:921080 [wandb_init.py:_resume_backend():245] resuming backend
2021-04-09 10:58:46,168 INFO    MainThread:921080 [wandb_init.py:_pause_backend():240] pausing backend
2021-04-09 10:58:50,792 INFO    MainThread:921080 [wandb_init.py:_resume_backend():245] resuming backend
2021-04-09 10:58:50,795 INFO    MainThread:921080 [wandb_init.py:_pause_backend():240] pausing backend
2021-04-09 10:59:31,119 INFO    MainThread:921080 [wandb_init.py:_resume_backend():245] resuming backend
2021-04-09 11:01:57,796 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting env: {}
2021-04-09 11:01:57,796 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-09 11:01:57,796 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '4e022d9a965c0e513287d998fbd6fd1d46056300'}
2021-04-09 11:01:57,796 INFO    MainThread:921080 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-09 11:01:57,796 INFO    MainThread:921080 [wandb_setup.py:_flush():69] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f456829c4c0>
2021-04-09 11:01:57,796 INFO    MainThread:921080 [wandb_init.py:_log_setup():319] Logging user logs to /home/ubuntu/evgeny/GEP-forecasts/EVERGI/wandb/run-20210409_110157-120v0kv9/logs/debug.log
2021-04-09 11:01:57,797 INFO    MainThread:921080 [wandb_init.py:_log_setup():320] Logging internal logs to /home/ubuntu/evgeny/GEP-forecasts/EVERGI/wandb/run-20210409_110157-120v0kv9/logs/debug-internal.log
2021-04-09 11:01:57,797 INFO    MainThread:921080 [wandb_init.py:init():352] calling init triggers
2021-04-09 11:01:57,797 INFO    MainThread:921080 [wandb_init.py:init():357] wandb.init called with sweep_config: {}
config: {'batch_size': 1500, 'architecture': 'RNN with forward lags for temporal', 'dataset': 'Columbia', 'epochs': 100, 'patience': 10}
2021-04-09 11:01:57,797 INFO    MainThread:921080 [wandb_init.py:init():372] re-initializing run, found existing run on stack: 1xm392uh
2021-04-09 11:01:57,801 INFO    MainThread:921080 [wandb_run.py:finish():1078] finishing run ujohn33/3days_forcast/1xm392uh
2021-04-09 11:01:57,802 INFO    MainThread:921080 [wandb_init.py:_jupyter_teardown():250] cleaning up jupyter logic
2021-04-09 11:01:57,840 INFO    MainThread:921080 [jupyter.py:save_history():237] saving 16 cells to _session_history.ipynb
2021-04-09 11:01:57,847 INFO    MainThread:921080 [wandb_run.py:_config_callback():669] config_cb None None {'batch_size': 1500, 'architecture': 'RNN with forward lags for temporal', 'dataset': 'Columbia', 'epochs': 100, 'patience': 10, '_wandb': {'session_history': 'code/_session_history.ipynb'}}
